filter/filter_impl/active_filter.go

```diff
@@ -17,14 +17,22 @@
 
 package filter_impl
 
+import (
+	"strconv"
+)
+
 import (
 	"github.com/apache/dubbo-go/common/extension"
 	"github.com/apache/dubbo-go/common/logger"
 	"github.com/apache/dubbo-go/filter"
 	"github.com/apache/dubbo-go/protocol"
+	invocation2 "github.com/apache/dubbo-go/protocol/invocation"
 )
 
-const active = "active"
+const (
+	active                  = "active"
+	dubbo_invoke_start_time = "dubbo_invoke_start_time"
```

Comment:
i think you should use camel case for the var

Reply:
good idea.